.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -24,7 +24,7 @@ locals {
   frontend_cmd                 = []
   # TODO: Assess whether this is safe for Portal API as well. Trying 1 worker in rdev portal backend containers, to minimize use of memory by TileDB (allocates multi-GB per process)
   # Note: keep-alive timeout should always be greater than the idle timeout of the load balancer (60 seconds)
-  backend_workers              = var.backend_cpus * 2 + 1 # Number of backend workers to run. Using the flask formula 2*cpu+1
+  backend_workers              = var.backend_workers 
```

Comment:
It would be cleaner to make this `var.backend_cpus + 1`. Then we don't need to add the `backend_worker` var, less hardcode variables, and consistently assigned number of workers.

Reply:
I'd like to fix dev and staging to 1 worker, though.

Reply:
In that case set the default of backend_workers to 1 and add a comment as to why it is 1.

Reply:
Gotcha